### ISSUE TITLE
Improve design of the landign page 

### ISSUE THREAD

**Issue by shubhagarwal1** [subhadeeproy3902/trailgo#84]
<img width="1432" alt="Screenshot 2024-10-21 at 5 19 30 AM" src="https://github.com/user-attachments/assets/c7426c4c-555c-4a06-869f-36cb5d7ab7e9">
For now our landing page is preety minilistic 
I propose to keep the current theme but add some new compoennts to the landing page 
1. crousel 
2. hoverable div cards
3. improve design for div cards 
4. Add more content to the page

**Comment by subhadeeproy3902** [comment]
Go ahead make use of framer motion and make a nice animated hero section ! with cool fonts too ! If u can !
An example -> 
![1](https://github.com/user-attachments/assets/da7d797d-a9d7-441c-bd95-942e80313ad2)

This is a Powerpoint Presentation not UI 
But I would love to have a animated hero section with the train move from left to right on scroll. Make a designer one and aesthetic one for this project !

**Comment by shubhagarwal1** [comment]
> Go ahead make use of framer motion and make a nice animated hero section ! with cool fonts too ! If u can ! An example -> ![1](https://private-user-images.githubusercontent.com/111780029/378462430-da7d797d-a9d7-441c-bd95-942e80313ad2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2ODkyMzYsIm5iZiI6MTcyOTY4ODkzNiwicGF0aCI6Ii8xMTE3ODAwMjkvMzc4NDYyNDMwLWRhN2Q3OTdkLWE5ZDctNDQxYy1iZDk1LTk0MmU4MDMxM2FkMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyM1QxMzA4NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YmQzMDgxOTIwZjYyNDZlMWJkYjM5ZjA2MTM0MmFlOTVhZTExZWVkZDFlNDQwYTE0YWQ4M2UxY2UxNDkxNTE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wW8dCg0vbM40wH9xpT0jiBynnH_-Wdv3SwxAPut4Phw)
> 
> This is a Powerpoint Presentation not UI But I would love to have a animated hero section with the train move from left to right on scroll. Make a designer one and aesthetic one for this project !

i will follow this

**Comment by Amruta7203** [comment]
Any updates @shubhagarwal1?

**Comment by shubhagarwal1** [comment]
> Any updates @shubhagarwal1?

yes the design is ready i will present it to you by today, will raise pull request by EOD

**Comment by subhadeeproy3902** [comment]
> > Any updates @shubhagarwal1?
> 
> yes the design is ready i will present it to you by today, will raise pull request by EOD

when ?
